OJP/OJP_FareSupport.xsd

```diff
@@ -697,15 +697,20 @@
 	</xs:group>
 	<xs:complexType name="TripFareResultStructure">
 		<xs:annotation>
-			<xs:documentation>Structure of a Single Trip Fare Request result</xs:documentation>
+			<xs:documentation>Structure of a single TripFareResult. This may cover the whole trip or only the part the service can answer. If only partial answers can be delivered, then this must be indicated in the error condition (as a warning). For a given trip multiple FareResults may need to be combined to get the whole tariff. A typical example may be classic public transport and a sharing leg. Some legs, e.g. TransferLegs and walking ContinuousLeg will never have a price. The service may include them in one of the tickets. If the whole trip is for free, then a TripFareResult with a price of 0 is expected. It is possible to have combined FareResults (e.g. for a bundle) and in addition price each leg individually.</xs:documentation>
```

Comment:
Do we need a addidion to the error conditions for the proposed warning?

Reply:
@ue71603 I would agree.

Reply:
`			<xs:enumeration value="FARE_ADDITIONALCHARGES">
				<xs:annotation>
					<xs:documentation>Passengers may be charged additional fees (e.g.: for road tolls or seat reservation).</xs:documentation>
				</xs:annotation>
			</xs:enumeration>
			<xs:enumeration value="FARE_ADDITIONALTICKETS">
				<xs:annotation>
					<xs:documentation>Additional tickets may be necessary because only parts of the passenger trip could be covered.</xs:documentation>
				</xs:annotation>
			</xs:enumeration>`

Reply:
Yes. FARE_ADDITIONALTICKETS should cover it. Maybe it would be good to explicitly name it here then instead of "must be indicated in the error condition (as a warning)"?